Documentation/Configuration/Typo3ConfVars/Index.rst

```diff
@@ -57,11 +57,11 @@ installations.
 This file overrides default settings from
 :file:`typo3/sysext/core/Configuration/DefaultConfiguration.php`.
 
-.. attention::
-
-   Since configuration settings can be manipulated from within the
-   TYPO3 backend, the :file:`config/system/settings.php`
-   must be writable by the web server user.
+..  versionchanged:: 12.1
```

Comment:
As a general rule I would write versionchanged directives in a way that they can be removed in one or two versions future. So the permanently important information should be outside of the versionchanged box